src/ipLoadBalancing/farm/ipLoadBalancing-farm-server.lexi.service.js

```diff
@@ -0,0 +1,41 @@
+"use strict";
+
+_.forEach(["tcp", "udp", "http"], function (type) {
+    angular
+        .module("ovh-api-services")
+        .service("OvhApiIpLoadBalancingFarm" + _.capitalize(type) + "ServerLexi",
+                 ["$resource", "$cacheFactory", function ($resource, $cacheFactory) {
+                     var cache = $cacheFactory("OvhApiIpLoadBalancingFarm" + _.capitalize(type) + "ServerLexi");
```

Comment:
Optimization : You should `_.capitalize` one time type, store the result in const and use it after ;)

Reply:
Template string ;)

Reply:
You should use `const` instead of `var`